index.js

```diff
@@ -179,6 +188,21 @@ function shortenURL(href, currentUrl = 'https://github.com') {
 		return `${partial}${search}${hash} (compare)`;
 	}
 
+	// Shorten URLs that would otherwise be natively shortened
+	if (isRedirection) {
+		if (issue) {
+			return `${repoUrl}#${issue}`;
+		}
+
+		if (pull) {
+			return `${repoUrl}#${pull}`;
+		}
+
+		if (commit) {
+			return `<code>${commit.slice(0, 7)}</code>`;
```

Comment:
This doesn't support cross-repo commit links, like

https://github.com/refined-github/refined-github/commit/4f270c4f50e0a2a20085a6e92095117f10340322